.storybook/webpack.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  resolve: {
+    alias: {
+      "react-impl": "react"
```

Comment:
So, I'm curious why this is necessary.

For the `storybook-preact` compilation, it seems like we should alias `react` to either `preact-compat` or `preact`, whichever we want to test against. That way the stories themselves don't have to change, and the regular webpack is a bit simpler. If we decide we want to test `preact-compat` instead of `preact`, we just change the value in `.storybook-preact/webpack.config.js`, instead of having to update all the stories.